code/modules/projectiles/guns/energy/shrapnel.dm

```diff
@@ -23,10 +23,10 @@
 		list(mode_name="stun", projectile_type=/obj/item/projectile/bullet/shotgun/beanbag, charge_cost=25, icon="stun"),
 		list(mode_name="blast", projectile_type=/obj/item/projectile/bullet/shotgun, charge_cost=null, icon="destroy"),
 	)
-	var/consume_cell = FALSE
+	damage_multiplier = 1.15
```

Comment:
?

Reply:
this is due to conflicts with the balance of guns.

Reply:
solved